src/screens/ProfileScreen/index.js

```diff
@@ -573,15 +573,8 @@ const ProfileScreen = ({route}) => {
     await downVote(post);
   };
 
-  const handleOnEndReached = () => {
-    if (!isLastPage) {
-      getMyFeeds(postOffset);
-    }
-  };
-
   function handleRefresh() {
     setLoading(true);
-    setIsLastPage(false);
     getMyFeeds(0, LIMIT_PROFILE_FEED);
     reloadFetchAnonymousPost();
     fetchMyProfile(true);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleRefresh` function now directly calls `getMyFeeds` with a hardcoded limit. This might be problematic if the limit changes in the future. Consider defining a constant for the limit at the top of your file or module and using that constant both here and wherever else the limit is needed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
closed